mmdet/core/bbox/assigners/max_iou_assigner.py

```diff
@@ -193,6 +193,10 @@ def assign_wrt_overlaps(self, overlaps, gt_labels=None):
             # This might be the reason that it is not used in ROI Heads.
             for i in range(num_gts):
                 if gt_max_overlaps[i] >= self.min_pos_iou:
+                    # `min_pos_iou` is set to avoid assigning bboxes with
```

Comment:
We should add this description in the docstring.
We can briefly say the improvements is about 0.3. For example
```
# `min_pos_iou` is set to avoid assigning bboxes that have extremely small iou with GT as positive sample.
#  It brings about 0.3 mAP improvements and more comparison can be find in xxx

```

Reply:
Done.